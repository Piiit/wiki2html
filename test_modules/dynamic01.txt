Variable x has the value "{{x}}"
