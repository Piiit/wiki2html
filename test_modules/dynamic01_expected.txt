<!doctype html>
<html>
<head>
<title>wiki2html: no title</title>
</head>
<body>

Variable x has the value "111".

Variable x has the value "ValueX".

</body>
</html>

